docs/backup_restore.md

```diff
@@ -0,0 +1,52 @@
+In this section, you'll learn how to create backups of the rke2 cluster data and to restore the cluster from backup.
+
+### Creating Snapshots
+
+Snapshots are enabled by default.
+
+The snapshot directory defaults to `/server/db/snapshots`.
+
+To configure the snapshot interval or the number of retained snapshots, refer to the [options.](#options)
+
+
+## Cluster Reset
+
+RKE2 enables a feature to reset the cluster to one member cluster by passing `--cluster-reset` flag, when passing this flag to rke2 server it will reset the cluster with the same data dir in place, the data directory for etcd exists in `/server/db/etcd`, this flag can be passed in the events of quorum loss in the cluster.
```

Comment:
```suggestion
RKE2 enables a feature to reset the cluster to one member cluster by passing `--cluster-reset` flag, when passing this flag to rke2 server it will reset the cluster with the same data dir in place, the data directory for etcd exists in `/var/lib/rancher/rke2/server/db/etcd`, this flag can be passed in the events of quorum loss in the cluster.
```